### ISSUE TITLE
BUG: Mobile GUI not working

### ISSUE THREAD

**Issue by Jcubby14** [05Konz/Blooket-Cheats#107]
### Script

Mobile gui

### Problem

When i put the mobile gui in it worked but then a few weeks later it stopped working.

### Method

Put 'javascript:(code)' in the url bar or a bookmark

### Type

Unobfuscated

### Extra Info

Other guis work idk about reactive tho

**Comment by BlooketGodFr** [comment]
Yeah I’m having l same issue yesterday the gui was working but now the gui won’t work at all it won’t show up and I also put JavaScript:(code)

**Comment by 05Konz** [comment]
Is this just with the minified scripts? Also I pushed a fix, reopen if it still doesn't work

**Comment by SoggyPineapple** [comment]
It still doesn’t work
